test/ex_doc/language/erlang_test.exs

```diff
@@ -9,45 +9,45 @@ defmodule ExDoc.Language.ErlangTest do
 
   describe "autolink_doc/2 for edoc" do
     test "module", c do
-      assert autolink_edoc("{@link erlang_bar}", c) ==
+      assert autolink_edoc("{@link erlang_bar}", c) =~
```

Comment:
@garazdawi I don't think we need this change and can keep the stricter assertion here and in similar spots, i.e. ensure the exact html output. I'd use `=~` only on error messages. I went ahead and push this change to your branch but lmk if I was missing something!

Reply:
Without that change, the tests break for OTP-27 as all edoc renderings return `<p>....</p>` instead of `...`.

Reply:
Got it!